src/components/VaccineCard/VaccineCard.test.js

```diff
@@ -21,23 +21,23 @@ const patientData = {
           coding: [
             {
               system: 'http://hl7.org/fhir/sid/cvx',
-              code: '208'
-            }
-          ]
+              code: '208',
```

Comment:
Do you have a linter that automatically adds these trailing commas? This test fixture comes directly from a FHIR JSON Bundle, so I don’t think it necessarily makes sense to add them here. In terms of interoperability, JSON doesn't allow trailing commas, so it's probably not worth adding trailing commas to any new JSON fixture that gets copied over to a Javascript object in our repo.